### ISSUE TITLE
Currency transaction goes out of balance

### ISSUE THREAD

**Issue by sweitmann** [ledger123/runmyaccounts#67]
Hi Armaghan,

This issue is related to https://github.com/ledger123/runmyaccounts/issues/59

I have posted a transaction on app.myleder.net with the reference "12345":

![bildschirmfoto vom 2016-06-06 14 24 12](https://cloud.githubusercontent.com/assets/100664/15822009/896841a2-2bf3-11e6-8dd5-caf1ee5aa06b.png)

If you try to open this transaction and change the currency from GPB to EUR and then re-posting it...

![bildschirmfoto vom 2016-06-06 14 25 38](https://cloud.githubusercontent.com/assets/100664/15822026/97b1477c-2bf3-11e6-8725-7385bf966b83.png)

...you will see that it goes out of balance in debit and credit. 

![bildschirmfoto vom 2016-06-06 14 26 48](https://cloud.githubusercontent.com/assets/100664/15822035/a7a77d7c-2bf3-11e6-86ac-d4a98e7c61b2.png)

Any ideas why?

Regards

**Comment by ledger123** [comment]
Hi Sebastian,

This is 100% rounding difference issue and not a bug. I tried changing code to calculate in difference ways but always had this difference. So we have two possibilities:

1. We calculate the difference (0.01) and add it to the other side to any other fx trans line. I have done this and pushed the code to github. This is invisible to user and probably no one is going to notice and probably not going to disturb any thing.

I have modified code and pushed with this trick.

2. We add the differnece as a new transaction line with any suitable account. If you think this is better way then I can modify the code accordingly.

Regards
